release/models/system/openconfig-system.yang

```diff
@@ -266,6 +272,17 @@ module openconfig-system {
         information such as the current system date and time, uptime,
         last login timestamp, etc.";
     }
+
+    leaf my-mac-address {
+      type oc-yang:mac-address;
+      description
+        "MAC address to be added to the my-mac-address table, in addition to
```

Comment:
Is there a reference to this table's existence generically across system implementations? If not, it would be worth defining what "my" means. I assume that you are specifying something that is treated as a local MAC owned by the controller cards of the device.

Reply:
+1 to move leaf to `/system/config/mac-address`  and also request a more elaborate description for this leaf.  The description should help answer questions like:

* are packets sourced by the controller card intended to use this mac address as a source mac address?
* are packets ingress to the device with a dest mac matching `/system/config/mac-address`  expected to be  punted to the controller card?  
* are there any other functions affected by this mac address?

What is a mac address table?  Do we need to refer or define a mac address table, and/or Is this meant to be an ARP table?

Reply:
Added clarification for these. Thanks.